jobs/mesoscale/stats/JEVS_MESOSCALE_STATS

```diff
@@ -29,32 +29,34 @@ echo "BEGIN: $(basename ${BASH_SOURCE[0]})"
 export PS4='$SECONDS + '
 date
 
-# LSF id (jobid) and Temp Directories
-export pid=$$
-export pgmout="OUTPUT.${pid}"
-
 export SENDCOM=${SENDCOM:-YES}
 export SENDDBN=${SENDDBN:-YES}
 export SENDECF=${SENDECF:-YES}
 
-export DATA=${DATA:-${DATAROOT:?}}/${jobid:?}
-rm -rf $DATA
+export DATA=${DATA:-${DATAROOT}/${jobid:?}
```

Comment:
@RoshanShrestha-NOAA , can you please update this line from:
`export DATA=${DATA:-${DATAROOT}/${jobid:?}`
to:
`export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}`

Reply:
The correction is made. Also, I have added 
export pid=$$
in 
jobs/mesoscale/plots/JEVS_MESOSCALE_PLOTS
and 
jobs/mesoscale/stats/JEVS_MESOSCALE_STATS
to avoid the job failure. Now my test runs are going good.